### PR TITLE
School Library: Feature/add setup

### PR DESCRIPTION
**Pull request by mateo951**
- Added readme file
- Added ruby linters
- Added person class
- Added student class
- Added teacher class
- Modified person class

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,31 @@
...
+
+The objective of this project is to practice Object Oriented Programming. The app will be a library that simulates the following actions:
+- Add new students or teachers.
+- Add new books.
+- Save records of who borrowed a given book and when.
+
+Built With
+
+- Ruby
+
```
Comment (SajjadAhmad14) [inline README.md:11]: Please add instructions to run the project locally for a non-technical user.

File: person.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    true if @age >= 18 || @parent_permission
+  end
+
+  def check_age?
+    of_age?
+  end
+end
+
+# person1 = Person.new(17)
+# puts person1.check_age?
```
Comment (SajjadAhmad14) [inline person.rb]: Please remove commented code from here as it can make the whole codebase unclean.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Mateo, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
Git Flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: person.rb (hunk 2 above)
Comment (mateo951) [reply person.rb]: Done ✔

File: README.md (hunk 1 above)
Comment (mateo951) [reply README.md:11]: Hi @SajjadAhmad14, a readme file is not even required in the specifications.

File: person.rb (hunk 2 above)
Comment (henatan99) [reply person.rb]: This issue has been addressed well. Nice job! :pray:

File: README.md (hunk 1 above)
Comment (henatan99) [reply README.md:11]: - [ ] Hi @mateo951 , a professional repo with descriptive readme is part of the general requirements. Kindly refer back to the [project's requirement documentation page](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md) and find the link to the [professional repo documentation page](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) under the "General Requirements" section. Kindly make changes here as per the request made by the previous reviewer. :pray:

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @mateo951 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Mateo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 :
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**